### ISSUE TITLE
Exclude elements linking to sections of a Wikipedia page from "wikipedia wikidata mismatch" check

### ISSUE THREAD

**Issue by ivanbranco** [matkoniecz/OSM-wikipedia-tag-validator-reports#2]
Czesc Mateusz,

wikipedia=* tag syntax allows to link sections of a wiki page with an hashtag: wikipedia=language:page title#section_of_the_page

I guess in these cases is implied that the element isn't notable enough to have an entire Wikipedia page for itself (while the Wikidata notability threshold is lower), so they could be treated as false positive. An example: https://www.openstreetmap.org/way/14116957

**Comment by ivanbranco** [comment]
> note that in such case Wikidata should link to redirect pointing to a section - see https://www.wikidata.org/wiki/Wikidata:Sitelinks_to_redirects
> 
> (not sure is tool smart enough to not complain in such case)

Uhm, I see. I edited accordingly (I think). I hope it's not something against Wikipedia italia policies. Also because I know different language versions have different policies, e.g. italian wikipedia has a way higher notability threshold than the english one.  

Thanks for the info, I didn't know about this. We will see if they will delete the redirects I created for those lakes.

**Comment by matkoniecz** [comment]
Yeah, obviously I am not setting policy on Italian Wikipedia and neither Wikidata community is doing this.

This is a bit irritating as Wikidata cache is not refreshed automatically, I will refresh this one manually and check whether my tool is still complaining.

Also, please let me know if it is against Italian Wikidata community guidelines (I am not setting them or able to override them, the same goes for Wikidata community)

**Comment by matkoniecz** [comment]
And in general I guess that this type of issue can be moved to own section anyway, distinct from "wikipedia wikidata mismatch" general check ( "wikipedia wikidata mismatch, wikipedia links to a section"?)
